xbmc/utils/CharsetConverter.cpp

```diff
@@ -462,7 +464,9 @@ bool CCharsetConverter::CInnerConverter::convert(iconv_t type, int multiplier, c
   return true;
 }
 
-bool CCharsetConverter::CInnerConverter::logicalToVisualBiDi(const std::u32string& stringSrc, std::u32string& stringDst, FriBidiCharType base /*= FRIBIDI_TYPE_LTR*/, const bool failOnBadString /*= false*/)
+bool CCharsetConverter::CInnerConverter::logicalToVisualBiDi(const std::u32string& stringSrc,
+  std::u32string& stringDst, FriBidiCharType base /*= FRIBIDI_TYPE_LTR*/,
+  const bool failOnBadString /*= false*/, int* logicalToVisualMap /*= nullptr*/)
```

Comment:
Like this. We need to use a 4 space continuance as aligning to the function brackets breaks the 100 char width limit.

```cpp
bool CCharsetConverter::CInnerConverter::logicalToVisualBiDi(
    const std::u32string& stringSrc,
    std::u32string& stringDst,
    FriBidiCharType base /*= FRIBIDI_TYPE_LTR*/,
    const bool failOnBadString /*= false*/,
    int* logicalToVisualMap /*= nullptr*/)
```